homeassistant/components/triggercmd/switch.py

```diff
@@ -0,0 +1,105 @@
+"""Platform for switch integration."""
+
+from __future__ import annotations
+
+import logging
+
+from triggercmd import client
+
+from homeassistant.components.switch import SwitchEntity
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import TriggercmdConfigEntry
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: TriggercmdConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Add switch for passed config_entry in HA."""
+    hub = config_entry.runtime_data.hub
+    async_add_entities(TRIGGERcmdSwitch(switch) for switch in hub.switches)
+
+
+class TRIGGERcmdSwitch(SwitchEntity):
```

Comment:
For new integrations, we require the following:
- Support for [the new entity naming](https://developers.home-assistant.io/docs/core/entity#has_entity_name-true-mandatory-for-new-integrations)
- Entities and their attributes should be [translatable](https://developers.home-assistant.io/blog/2023/03/27/entity_name_translations/)

Please adopt your code to add support for it.

Reply:
I think I resolved the new entity naming issue.  
I'm not sure what to do about making the entities translatable because their names are based on TRIGGERcmd user data.
I need to do something, I'll try to figure it out, but I'd appreciate any clues you can give me.